docs/sources/administration/image_rendering.md

```diff
@@ -7,7 +7,9 @@ weight = 300
 
 # Image rendering
 
-Grafana supports automatic rendering of panels and dashboards as PNG images. This allows Grafana to automatically generate images of your panels to include in [alert notifications]({{< relref "../alerting/notifications.md" >}}).
+Grafana supports automatic rendering of panels as PNG images. This allows Grafana to automatically generate images of your panels to include in [alert notifications]({{< relref "../alerting/notifications.md" >}}).
+
+>**Note:** Image rendering of dashboards is not supported at this time.
```

Comment:
Not something we officially support, but definitely possible even though not supported via UI - you have to manually browse `/render/d/<uid>/slug?<render parameters>`. Depending on dashboard it could work or not work why it's unofficial. Not sure this is something we want to document since it's unofficial? 

Seems like the issue referenced is asking for official support and being able to do it thru the UI. 

Based on this information would you like to change anything?

Reply:
Thank you @marefr for the detailed explanation. We typically would not document non-supported features. 
Let us keep the changes made in the PR. If the feature is supported in the future, we will update the docs.